nicegui/elements/chart.js

```diff
@@ -8,7 +8,10 @@ export default {
   },
   methods: {
     update_chart() {
-      this.chart.update(this.options);
+      while (this.chart.series.length > this.options.series.length) this.chart.series[0].remove();
+      while (this.chart.series.length < this.options.series.length) this.chart.addSeries({}, false);
+      this.chart.update(this.options, false);
```

Comment:
Is there a reason for calling `update(..., redraw=false)` right before calling `redraw()`?

Reply:
Ah, good point. I simplified the code in the commit below.